src/genetic_forensic_portal/app/pages/5_Get_Analysis_Status.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+import streamlit as st
+
+from genetic_forensic_portal.app.client import gf_api_client as client
+
+st.title("Analysis Status Checker")
+
+uuid = st.selectbox(
+    "Select a sample ID",
+    client.list_completed_analyses(),
+    index=None,
+    placeholder="Select sample ID...",
+)
+
+if uuid:
+    try:
+        status = client.get_analysis_status(uuid)
+        st.success(f"Current status of the analysis: {status}")
+    except ValueError as ve:
+        st.error(f"Invalid input: {ve}")
+    except FileNotFoundError:
+        st.error("Analysis not found for the given UUID")
+    except Exception as e:
+        st.error(f"An unexpected error occurred: {e!s}")
```

Comment:
I think this type conversion may be unnecessary

```suggestion
        st.error(f"An unexpected error occurred: {e}")
```